Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -98,6 +98,22 @@ public extension Node where Context: HTMLNamableContext {
     }
 }
 
+public extension Attribute where Context == HTML.MetaContext {
+    /// Assign a property to the element.
+    /// - parameter property: The property to assign.
+    static func property(_ property: String) -> Attribute {
+        Attribute(name: "property", value: property)
+    }
+}
+
+public extension Node where Context: HTMLNamableContext {
```

Comment:
I don't think this is needed, considering that the `meta` API only accepts `Attribute` instances, not nodes. So, unless I'm missing something, I think you can safely remove this.